CODEOWNERS

```diff
@@ -115,7 +110,6 @@ Makefile* @cilium/build
 /Documentation/gettingstarted/kubeproxy-free.rst @cilium/loadbalancer @cilium/docs-structure
 /Documentation/gettingstarted/policy-creation.rst @cilium/policy @cilium/docs-structure
 /Documentation/glossary.rst @cilium/docs-structure
-/Documentation/hubble.rst @cilium/hubble @cilium/docs-structure
```

Comment:
This was renamed to `Documentation/internals/hubble.rst` in db9b6f71453cdb99253ce22a586d0a7e5b66322e, so probably better to change the file path here instead of deleting it.